include/rtc/description.hpp

```diff
@@ -50,6 +50,7 @@ class RTC_CPP_EXPORT Description {
 	string bundleMid() const;
 	std::vector<string> iceOptions() const;
 	optional<string> iceUfrag() const;
+
```

Comment:
It seems this white line was added by mistake.

Reply:
Fixed!